programs/editor/Tools.js

```diff
@@ -105,9 +105,12 @@ define("webodf/editor/Tools", [
                 sessionSubscribers.forEach(function (subscriber) {
                     subscriber.setEditorSession(editorSession);
                 });
-                if (formatMenuButton) {
-                    formatMenuButton.setAttribute('disabled', !editorSession);
-                }
+
+                [saveButton, saveAsButton, downloadButton, closeButton, formatMenuButton].forEach(function (button) {
```

Comment:
I see. But that is an independent error, no? No proper handling of failed loading of documents yet :/